media/common/src/main/java/io/helidon/media/common/MediaContext.java

```diff
@@ -160,6 +163,11 @@ private Builder() {
          * @param config a {@link Config}
          * @return this {@link Builder}
          */
+        @ConfiguredOption(key = "services",
```

Comment:
As far as I remember, the services works same as security providers, so it should use the same style:
`@ConfiguredOption(key = "services", kind = ConfiguredOption.Kind.LIST, required = true, provider = true)`

in your case, you need to explicitly define the type as the class of the service provider interface (if it is not on a method that adds it - in this case `MediaSupportProvider`)